src/utils/time.ts

```diff
@@ -52,18 +68,22 @@ const getDates = (start: string, end: string): ContestSchedule => {
     botRaceEnd
   );
 
+  const currentCohort = getCurrentCohortDates(cohorts);
+
   return {
     contestStatus,
     botRaceStatus,
     start: startDate,
     end: endDate,
+    pause: currentCohort.pauseDate,
+    resume: currentCohort.resumeDate,
     botRaceEnd,
     formattedEnd: format(endDate, "d MMM h:mm a"),
     formattedStart: format(startDate, "d MMM h:mm a"),
     timeZone: timeZone,
-    formattedBotRaceEnd: format(botRaceEnd, "d MMMM h:mm a"),
+    formattedBotRaceEnd: format(botRaceEnd, "d MMM h:mm a"),
```

Comment:
This was formatted differently, which I noticed in the tests. We don't have bot races anymore, but I figured these should match the other date formats.